custom-words.txt

```diff
@@ -3977,3 +3977,16 @@ protobuf
 proto
 msvideo
 SOQL
+webappcollectors
+wacc
+hyperthread
+Hyperthread
+Datacentre
```

Comment:
Please spell it datacenter, not datacentre

Reply:
Datacentre is being used in an enum right now. Hence the service understands Datacentre instead of Datacenter. This is already in production, hence we cannot take this change in the current version.

Reply:
Then how about please adding it in the cSpell exceptions list instead of in custom-words - since I still consider it a misspelling, and wouldn't want other RPs to prefer to use it.

Reply:
'webappcollectors' might be better in cSpell also. It doesn't look like the kind of term other RPs will want to reuse, to me.